test/test_helper.rb

```diff
@@ -9,3 +9,36 @@
 Dir["test/fixtures/**/*.rb"].each { |f| require File.expand_path(f) }
 
 Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+# Added assertions
+
+def assert_executes(exactly: nil, at_least: nil, message: "The given block did not execute the desired statement")
```

Comment:
Sure. The `assert_executes` wasn't something I intended to distribute as part of Slayer to end users. It was more intended as an internal helper for our own unit tests. I think what goes into the Slayer::Minitest and Slayer::RSpec (and what we ultimately are distributing as something target project's can include) should be helpers specific to Slayer. `assert_executes` really isn't specific to Slayer, just something I saw us doing a lot in our tests.

I agree we should move this to a module, but I think it should be an internal module of some kind.